plugins/spark/src/main/java/org/radargun/service/demo/ispn/WordCountSource.java

```diff
@@ -0,0 +1,55 @@
+package org.radargun.service.demo.ispn;
+
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.infinispan.spark.rdd.InfinispanJavaRDD;
+import org.radargun.service.SparkMapReduce;
+
+import java.util.Properties;
+
+import static java.util.Arrays.stream;
+import static java.util.stream.Collectors.toList;
+
+/**
+ * @author Matej Cimbora
+ */
+public class WordCountSource implements SparkMapReduce.SparkJavaRDDSource<String> {
```

Comment:
Splitting class into one abstract class with one abstract method and concrete with just the business method is overly complicated?